README.md

```diff
@@ -38,3 +38,71 @@ const formData = objectToFormData(
 
 console.log(formData);
 ```
+
+Practical ES6 Vue Axios example for PHP (and others). If you would like to receive data 
+in proper associate array format set option incides to true.
+```vue
+<template>
+
+    <div>
+        {{uploading ? 'UPLOADING' : ''}}
+        <button @click="sendMessage">Send message</button>
+    </div>
+
+</template>
+
+<script>
+
+    import axios from 'axios';
+    import objectToFormData from 'object-to-formdata';
+
+    export default {
+
+        methods: {
+            data() {
+                return {
+                    uploading: false
+                }
+            },
+            sendMessage() {
+                import objectToFormData from 'object-to-formdata';
```

Comment:
Why import `objectToFormData` twice?

Reply:
Must be a typo :-) Import at the top should remain.